docs/android.md

```diff
@@ -153,7 +153,7 @@ those sections before configuring and building OpenDDS.
 
 ```Shell
 ./configure --doc-group --target=android --macros=ANDROID_ABI=armeabi-v7a
-    --macros=ANDROID_API_PATH=$SDK/platforms/android-$API
+    --macros=android_sdk=$SDK --macros=android_target_api=$API
```

Comment:
This is just a minimal example and doesn't do anything because `--java` isn't being passed. If you want to keep it I would suggest moving this to the "Network Permissions and Availability" section.